ironmon_tracker/Utils.lua

```diff
@@ -39,3 +39,19 @@ function Utils.isSTAB(move, pkmnData)
 	end
 	return false
 end
+
+function Utils.calculateLowKickDamage(weight)
```

Comment:
Please change the name of this function to something more generic, since it can also be used with Grass Knot in Gen 4.